pkg/api/config_test.go

```diff
@@ -20,8 +20,9 @@ func TestValidate(t *testing.T) {
 				InputConfiguration: dummyInput,
```

Comment:
We should break up TestValidate to one test per validation func we have in the validate.go